R/pct_for_age_generic.R

```diff
@@ -40,19 +40,25 @@ pct_for_age_generic <- function(age = NULL, value = NULL, sex = NULL, variable="
     p[1] <- p[1]/10 # 0.1
     p[length(p)] <- p[length(p)]/10 # 99.9
     p_txt <- paste0("pct_", p)
-    if(value > max(tmp)) {
+    if(value >= max(tmp)) {
       message(paste0("Specified ", variable," > 99.9th percentile!"))
       pct <- list(percentile = 99.9)
     }
-    if(value < min(tmp)) {
+    if(value <= min(tmp)) {
       message(paste0("Specified ", variable, " < 0.1th percentile!"))
```

Comment:
```suggestion
    if(value <= min(tmp)) {
      message(paste0("Specified ", variable, " <= 0.1th percentile!"))
```
should we also update the message accordingly?